app/assets/locales/en.json

```diff
@@ -377,6 +375,32 @@
     "resend_btn_lbl": "Resend Verification"
   },
   "forms": {
+    "validations": {
```

Comment:
Validations are generic and context based.
So many forms can reuse the same validations messages (usually).
Please use `forms.validations.[field_name]` instead,